### DISCUSSION TITLE
Corruption of in-memory data detected - XFS

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#6761] [General] [answered]
https://github.com/Security-Onion-Solutions/securityonion/discussions/3203

I'm facing similar issue and reconfiguring it every time. Any solution on this?

**Comment by Jackson-Pollock** [comment]
![image](https://user-images.githubusercontent.com/62762611/148234988-d1697c35-8d42-4012-b11d-eabf7645919d.png)
error snap

**Comment by TOoSmOotH** [comment, marked as answer]
xfs_repair needs to be run on the file system in question. 

https://www.thegeekdiary.com/running-repairs-on-xfs-filesystems/
